2025-03-17 00:42:13,096 INFO    StreamThr :30328 [internal.py:wandb_internal():86] W&B internal server running at pid: 30328, started at: 2025-03-17 00:42:13.076650
2025-03-17 00:42:13,111 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status
2025-03-17 00:42:13,189 INFO    WriterThread:30328 [datastore.py:open_for_write():85] open: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\run-qh0l59uo.wandb
2025-03-17 00:42:13,191 DEBUG   SenderThread:30328 [sender.py:send():382] send: header
2025-03-17 00:42:13,281 DEBUG   SenderThread:30328 [sender.py:send():382] send: run
2025-03-17 00:42:14,018 INFO    SenderThread:30328 [dir_watcher.py:__init__():211] watching files in: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files
2025-03-17 00:42:14,018 INFO    SenderThread:30328 [sender.py:_start_run_threads():1136] run started: qh0l59uo with start time 1742152333.068651
2025-03-17 00:42:14,071 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: check_version
2025-03-17 00:42:14,073 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: check_version
2025-03-17 00:42:14,232 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: run_start
2025-03-17 00:42:14,262 DEBUG   HandlerThread:30328 [system_info.py:__init__():32] System info init
2025-03-17 00:42:14,262 DEBUG   HandlerThread:30328 [system_info.py:__init__():47] System info init done
2025-03-17 00:42:14,262 INFO    HandlerThread:30328 [system_monitor.py:start():194] Starting system monitor
2025-03-17 00:42:14,263 INFO    SystemMonitor:30328 [system_monitor.py:_start():158] Starting system asset monitoring threads
2025-03-17 00:42:14,263 INFO    HandlerThread:30328 [system_monitor.py:probe():214] Collecting system info
2025-03-17 00:42:14,273 INFO    SystemMonitor:30328 [interfaces.py:start():190] Started cpu monitoring
2025-03-17 00:42:14,274 INFO    SystemMonitor:30328 [interfaces.py:start():190] Started disk monitoring
2025-03-17 00:42:14,275 INFO    SystemMonitor:30328 [interfaces.py:start():190] Started gpu monitoring
2025-03-17 00:42:14,283 DEBUG   HandlerThread:30328 [system_info.py:probe():196] Probing system
2025-03-17 00:42:14,305 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:14,306 INFO    SystemMonitor:30328 [interfaces.py:start():190] Started memory monitoring
2025-03-17 00:42:14,309 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:14,319 INFO    SystemMonitor:30328 [interfaces.py:start():190] Started network monitoring
2025-03-17 00:42:14,348 DEBUG   HandlerThread:30328 [system_info.py:_probe_git():181] Probing git
2025-03-17 00:42:14,624 DEBUG   HandlerThread:30328 [system_info.py:_probe_git():189] Probing git done
2025-03-17 00:42:14,624 DEBUG   HandlerThread:30328 [system_info.py:probe():244] Probing system done
2025-03-17 00:42:14,624 DEBUG   HandlerThread:30328 [system_monitor.py:probe():223] {'os': 'Windows-10-10.0.26100-SP0', 'python': '3.9.5', 'heartbeatAt': '2025-03-16T19:12:14.306529', 'startedAt': '2025-03-16T19:12:12.970254', 'docker': None, 'cuda': None, 'args': (), 'state': 'running', 'program': '<python with no main file>', 'codePathLocal': None, 'git': {'remote': 'git@github.com-work:saimanikumar-da24m016/da6401_assignment1.git', 'commit': 'e828d7ed826d245b35090fa330e5d4f07dddcfac'}, 'email': 'saimanikumar67@gmail.com', 'root': 'C:/Users/D. Sai Mani Kumar/Downloads/DL/da6401_assignment1', 'host': 'LAPTOP-VRI27UL2', 'username': 'D. Sai Mani Kumar', 'executable': 'c:\\Users\\D. Sai Mani Kumar\\AppData\\Local\\Programs\\Python\\Python39\\python.exe', 'cpu_count': 4, 'cpu_count_logical': 8, 'cpu_freq': {'current': 2400.0, 'min': 0.0, 'max': 2400.0}, 'cpu_freq_per_core': [{'current': 2400.0, 'min': 0.0, 'max': 2400.0}], 'disk': {'/': {'total': 375.82519149780273, 'used': 272.6003837585449}}, 'gpu': 'NVIDIA GeForce GTX 1650 Ti', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce GTX 1650 Ti', 'memory_total': 4294967296}], 'memory': {'total': 7.840869903564453}}
2025-03-17 00:42:14,625 INFO    HandlerThread:30328 [system_monitor.py:probe():224] Finished collecting system info
2025-03-17 00:42:14,625 INFO    HandlerThread:30328 [system_monitor.py:probe():227] Publishing system info
2025-03-17 00:42:14,625 DEBUG   HandlerThread:30328 [system_info.py:_save_pip():52] Saving list of pip packages installed into the current environment
2025-03-17 00:42:14,629 DEBUG   HandlerThread:30328 [system_info.py:_save_pip():68] Saving pip packages done
2025-03-17 00:42:14,631 INFO    HandlerThread:30328 [system_monitor.py:probe():229] Finished publishing system info
2025-03-17 00:42:14,632 DEBUG   SenderThread:30328 [sender.py:send():382] send: files
2025-03-17 00:42:14,632 INFO    SenderThread:30328 [sender.py:_save_file():1392] saving file wandb-metadata.json with policy now
2025-03-17 00:42:14,679 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: stop_status
2025-03-17 00:42:14,680 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: stop_status
2025-03-17 00:42:14,690 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-17 00:42:14,963 DEBUG   SenderThread:30328 [sender.py:send():382] send: telemetry
2025-03-17 00:42:14,972 DEBUG   SenderThread:30328 [sender.py:send():382] send: telemetry
2025-03-17 00:42:14,973 DEBUG   SenderThread:30328 [sender.py:send():382] send: config
2025-03-17 00:42:14,973 DEBUG   SenderThread:30328 [sender.py:send():382] send: config
2025-03-17 00:42:14,973 DEBUG   SenderThread:30328 [sender.py:send():382] send: config
2025-03-17 00:42:14,973 DEBUG   SenderThread:30328 [sender.py:send():382] send: config
2025-03-17 00:42:14,973 DEBUG   SenderThread:30328 [sender.py:send():382] send: config
2025-03-17 00:42:14,974 DEBUG   SenderThread:30328 [sender.py:send():382] send: config
2025-03-17 00:42:14,974 DEBUG   SenderThread:30328 [sender.py:send():382] send: config
2025-03-17 00:42:14,974 DEBUG   SenderThread:30328 [sender.py:send():382] send: config
2025-03-17 00:42:14,974 DEBUG   SenderThread:30328 [sender.py:send():382] send: config
2025-03-17 00:42:15,032 INFO    Thread-45 :30328 [dir_watcher.py:_on_file_created():271] file/dir created: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\requirements.txt
2025-03-17 00:42:15,033 INFO    Thread-45 :30328 [dir_watcher.py:_on_file_created():271] file/dir created: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\wandb-metadata.json
2025-03-17 00:42:15,034 INFO    Thread-45 :30328 [dir_watcher.py:_on_file_created():271] file/dir created: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\output.log
2025-03-17 00:42:15,964 INFO    wandb-upload_0:30328 [upload_job.py:push():131] Uploaded file C:\Users\D9110~1.SAI\AppData\Local\Temp\tmpk9j8kduxwandb\kscrybh3-wandb-metadata.json
2025-03-17 00:42:17,684 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:17,685 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:17,706 INFO    Thread-45 :30328 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\output.log
2025-03-17 00:42:18,725 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:42:20,193 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:20,194 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:22,716 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:22,721 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:23,925 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:42:25,167 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:25,168 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:27,644 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:27,645 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:28,982 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:42:29,696 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-17 00:42:29,708 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: stop_status
2025-03-17 00:42:29,742 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: stop_status
2025-03-17 00:42:30,082 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:30,083 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:32,493 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:32,493 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:34,075 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:42:34,924 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:34,924 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:37,341 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:37,341 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:39,107 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:42:39,733 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:39,733 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:42,150 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:42,151 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:44,145 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:42:44,578 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:44,579 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:44,687 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-17 00:42:44,720 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: stop_status
2025-03-17 00:42:44,721 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: stop_status
2025-03-17 00:42:45,385 INFO    Thread-45 :30328 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\config.yaml
2025-03-17 00:42:47,000 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:47,001 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:49,431 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:49,432 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:50,122 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:42:50,126 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: partial_history
2025-03-17 00:42:50,128 DEBUG   SenderThread:30328 [sender.py:send():382] send: history
2025-03-17 00:42:50,128 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: summary_record
2025-03-17 00:42:50,131 INFO    SenderThread:30328 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-17 00:42:50,530 INFO    Thread-45 :30328 [dir_watcher.py:_on_file_created():271] file/dir created: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\wandb-summary.json
2025-03-17 00:42:52,600 INFO    Thread-45 :30328 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\output.log
2025-03-17 00:42:54,398 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:54,399 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:55,207 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:42:56,865 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:56,865 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:59,268 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:59,269 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:42:59,693 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-17 00:42:59,724 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: stop_status
2025-03-17 00:42:59,725 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: stop_status
2025-03-17 00:43:01,088 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:43:01,679 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:01,679 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:04,082 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:04,083 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:06,135 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:43:06,478 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:06,478 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:08,904 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:08,904 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:10,356 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: partial_history
2025-03-17 00:43:10,357 DEBUG   SenderThread:30328 [sender.py:send():382] send: history
2025-03-17 00:43:10,357 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: summary_record
2025-03-17 00:43:10,360 INFO    SenderThread:30328 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-17 00:43:10,943 INFO    Thread-45 :30328 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\wandb-summary.json
2025-03-17 00:43:11,315 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:11,315 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:11,362 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:43:12,988 INFO    Thread-45 :30328 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\output.log
2025-03-17 00:43:13,716 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:13,717 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:14,349 DEBUG   SystemMonitor:30328 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2025-03-17 00:43:14,353 DEBUG   SenderThread:30328 [sender.py:send():382] send: stats
2025-03-17 00:43:14,709 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-17 00:43:14,739 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: stop_status
2025-03-17 00:43:14,740 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: stop_status
2025-03-17 00:43:16,119 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:16,120 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:17,133 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:43:18,539 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:18,540 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:20,939 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:20,939 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:22,152 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:43:23,348 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:23,349 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:25,774 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:25,774 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:27,208 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:43:29,403 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: partial_history
2025-03-17 00:43:29,404 DEBUG   SenderThread:30328 [sender.py:send():382] send: history
2025-03-17 00:43:29,405 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: summary_record
2025-03-17 00:43:29,406 INFO    SenderThread:30328 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-17 00:43:29,721 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-17 00:43:29,770 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: stop_status
2025-03-17 00:43:29,771 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: stop_status
2025-03-17 00:43:30,241 INFO    Thread-45 :30328 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\wandb-summary.json
2025-03-17 00:43:30,702 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:30,702 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:31,276 INFO    Thread-45 :30328 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\output.log
2025-03-17 00:43:33,130 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:33,131 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:33,145 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:43:35,526 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:35,526 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:37,922 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:37,922 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:38,188 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:43:40,326 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:40,326 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:42,723 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:42,723 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:43,226 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:43:44,361 DEBUG   SenderThread:30328 [sender.py:send():382] send: stats
2025-03-17 00:43:44,723 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-17 00:43:44,784 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: stop_status
2025-03-17 00:43:44,785 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: stop_status
2025-03-17 00:43:45,132 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:45,133 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:45,876 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: partial_history
2025-03-17 00:43:45,877 DEBUG   SenderThread:30328 [sender.py:send():382] send: history
2025-03-17 00:43:45,877 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: summary_record
2025-03-17 00:43:45,878 INFO    SenderThread:30328 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-17 00:43:46,518 INFO    Thread-45 :30328 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\wandb-summary.json
2025-03-17 00:43:46,519 INFO    Thread-45 :30328 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\output.log
2025-03-17 00:43:47,567 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:47,568 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:48,889 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:43:49,979 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:49,979 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:52,385 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:52,386 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:53,925 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:43:54,777 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:54,777 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:57,174 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:57,174 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:58,967 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:43:59,565 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:59,565 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:43:59,745 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-17 00:43:59,786 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: stop_status
2025-03-17 00:43:59,787 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: stop_status
2025-03-17 00:44:01,961 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:01,961 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:02,924 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: partial_history
2025-03-17 00:44:02,926 DEBUG   SenderThread:30328 [sender.py:send():382] send: history
2025-03-17 00:44:02,926 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: summary_record
2025-03-17 00:44:02,928 INFO    SenderThread:30328 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-17 00:44:03,832 INFO    Thread-45 :30328 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\wandb-summary.json
2025-03-17 00:44:04,976 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:44:05,013 INFO    Thread-45 :30328 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\output.log
2025-03-17 00:44:07,095 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:07,096 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:09,501 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:09,502 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:10,003 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:44:11,926 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:11,927 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:14,362 DEBUG   SenderThread:30328 [sender.py:send():382] send: stats
2025-03-17 00:44:14,363 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:14,365 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:14,740 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-17 00:44:14,800 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: stop_status
2025-03-17 00:44:14,801 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: stop_status
2025-03-17 00:44:15,075 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:44:16,814 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:16,815 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:19,251 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:19,252 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:20,224 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:44:21,698 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:21,699 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:24,125 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:24,125 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:25,229 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:44:26,572 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:26,573 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:28,998 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:28,999 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:29,739 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-17 00:44:29,802 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: stop_status
2025-03-17 00:44:29,802 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: stop_status
2025-03-17 00:44:31,092 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:44:31,421 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:31,422 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:33,839 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:33,840 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:36,131 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:44:36,255 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:36,256 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:38,686 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:38,686 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:40,780 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: partial_history
2025-03-17 00:44:40,781 DEBUG   SenderThread:30328 [sender.py:send():382] send: history
2025-03-17 00:44:40,782 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: summary_record
2025-03-17 00:44:40,788 INFO    SenderThread:30328 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-17 00:44:41,714 INFO    Thread-45 :30328 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\wandb-summary.json
2025-03-17 00:44:41,791 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:44:43,731 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:43,733 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:43,910 INFO    Thread-45 :30328 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\output.log
2025-03-17 00:44:44,378 DEBUG   SenderThread:30328 [sender.py:send():382] send: stats
2025-03-17 00:44:44,752 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-17 00:44:44,822 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: stop_status
2025-03-17 00:44:44,823 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: stop_status
2025-03-17 00:44:46,181 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:46,182 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:47,332 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:44:48,629 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:48,631 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:51,056 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:51,057 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:52,425 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:44:53,464 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:53,466 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:55,855 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:55,855 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:57,460 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:44:58,249 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:58,250 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:44:59,761 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-17 00:44:59,839 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: stop_status
2025-03-17 00:44:59,839 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: stop_status
2025-03-17 00:45:00,641 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:00,641 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:03,040 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:03,040 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:03,210 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:45:05,434 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:05,435 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:07,847 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:07,847 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:08,251 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:45:10,247 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:10,248 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:12,644 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:12,644 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:13,301 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:45:14,062 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: partial_history
2025-03-17 00:45:14,063 DEBUG   SenderThread:30328 [sender.py:send():382] send: history
2025-03-17 00:45:14,063 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: summary_record
2025-03-17 00:45:14,067 INFO    SenderThread:30328 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-17 00:45:14,389 DEBUG   SenderThread:30328 [sender.py:send():382] send: stats
2025-03-17 00:45:14,550 INFO    Thread-45 :30328 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\wandb-summary.json
2025-03-17 00:45:14,766 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-17 00:45:14,844 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: stop_status
2025-03-17 00:45:14,844 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: stop_status
2025-03-17 00:45:15,035 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:15,035 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:15,706 INFO    Thread-45 :30328 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\output.log
2025-03-17 00:45:19,265 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:45:19,990 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:19,990 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:22,389 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:22,390 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:24,306 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:45:24,808 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:24,808 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:27,195 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:27,195 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:29,347 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:45:29,599 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:29,600 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:29,776 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-17 00:45:29,853 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: stop_status
2025-03-17 00:45:29,854 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: stop_status
2025-03-17 00:45:32,005 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:32,005 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:34,417 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:34,419 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:35,186 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:45:36,825 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:36,826 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:39,218 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:39,218 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:40,232 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:45:41,608 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:41,608 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:44,013 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:44,014 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:44,404 DEBUG   SenderThread:30328 [sender.py:send():382] send: stats
2025-03-17 00:45:44,780 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-17 00:45:44,858 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: stop_status
2025-03-17 00:45:44,859 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: stop_status
2025-03-17 00:45:46,242 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:45:46,447 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:46,450 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:48,254 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: partial_history
2025-03-17 00:45:48,254 DEBUG   SenderThread:30328 [sender.py:send():382] send: history
2025-03-17 00:45:48,255 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: summary_record
2025-03-17 00:45:48,256 INFO    SenderThread:30328 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-17 00:45:48,374 INFO    Thread-45 :30328 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\wandb-summary.json
2025-03-17 00:45:48,901 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:48,901 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:49,377 INFO    Thread-45 :30328 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\output.log
2025-03-17 00:45:51,281 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:45:51,328 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:51,328 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:56,351 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:45:56,353 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:56,353 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:58,767 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:58,767 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:45:59,796 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-17 00:45:59,859 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: stop_status
2025-03-17 00:45:59,860 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: stop_status
2025-03-17 00:46:01,157 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:01,157 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:02,225 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:46:03,561 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:03,561 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:05,961 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:05,962 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:07,260 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:46:08,370 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:08,370 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:10,779 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:10,828 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:12,302 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:46:13,240 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:13,240 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:14,410 DEBUG   SenderThread:30328 [sender.py:send():382] send: stats
2025-03-17 00:46:14,805 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-17 00:46:14,898 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: stop_status
2025-03-17 00:46:14,899 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: stop_status
2025-03-17 00:46:15,634 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:15,634 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:18,036 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:18,036 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:18,304 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:46:19,060 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: partial_history
2025-03-17 00:46:19,062 DEBUG   SenderThread:30328 [sender.py:send():382] send: history
2025-03-17 00:46:19,062 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: summary_record
2025-03-17 00:46:19,063 INFO    SenderThread:30328 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-17 00:46:19,967 INFO    Thread-45 :30328 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\wandb-summary.json
2025-03-17 00:46:19,967 INFO    Thread-45 :30328 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\output.log
2025-03-17 00:46:20,435 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:20,436 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:22,855 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:22,855 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:24,098 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:46:25,257 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:25,257 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:27,651 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:27,651 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:29,138 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:46:29,820 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-17 00:46:29,883 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: stop_status
2025-03-17 00:46:29,883 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: stop_status
2025-03-17 00:46:32,511 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:32,511 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:34,278 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:46:34,905 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:34,906 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:37,288 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:37,288 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:39,308 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:46:39,686 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:39,686 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:42,078 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:42,078 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:44,348 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:46:44,413 DEBUG   SenderThread:30328 [sender.py:send():382] send: stats
2025-03-17 00:46:44,475 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:44,476 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:44,836 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-17 00:46:44,897 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: stop_status
2025-03-17 00:46:44,898 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: stop_status
2025-03-17 00:46:45,916 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: partial_history
2025-03-17 00:46:45,918 DEBUG   SenderThread:30328 [sender.py:send():382] send: history
2025-03-17 00:46:45,919 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: summary_record
2025-03-17 00:46:45,921 INFO    SenderThread:30328 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-17 00:46:46,184 INFO    Thread-45 :30328 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\wandb-summary.json
2025-03-17 00:46:46,887 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:46,887 ERROR   gpu       :30328 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2025-03-17 00:46:46,973 DEBUG   SenderThread:30328 [sender.py:send():382] send: files
2025-03-17 00:46:46,973 INFO    SenderThread:30328 [sender.py:_save_file():1392] saving file media\images\Confusion Matrix_10_0922b0f1b30ffc8f26da.png with policy now
2025-03-17 00:46:46,975 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: partial_history
2025-03-17 00:46:46,976 DEBUG   SenderThread:30328 [sender.py:send():382] send: history
2025-03-17 00:46:46,976 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: summary_record
2025-03-17 00:46:46,977 INFO    SenderThread:30328 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-17 00:46:47,201 INFO    Thread-45 :30328 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\wandb-summary.json
2025-03-17 00:46:47,201 INFO    Thread-45 :30328 [dir_watcher.py:_on_file_created():271] file/dir created: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\media\images\Confusion Matrix_10_0922b0f1b30ffc8f26da.png
2025-03-17 00:46:47,204 INFO    Thread-45 :30328 [dir_watcher.py:_on_file_created():271] file/dir created: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\media
2025-03-17 00:46:47,204 INFO    Thread-45 :30328 [dir_watcher.py:_on_file_created():271] file/dir created: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\media\images
2025-03-17 00:46:47,329 DEBUG   SenderThread:30328 [sender.py:send():382] send: telemetry
2025-03-17 00:46:47,336 DEBUG   SenderThread:30328 [sender.py:send():382] send: exit
2025-03-17 00:46:47,337 INFO    SenderThread:30328 [sender.py:send_exit():589] handling exit code: 0
2025-03-17 00:46:47,337 INFO    SenderThread:30328 [sender.py:send_exit():591] handling runtime: 273
2025-03-17 00:46:47,338 INFO    SenderThread:30328 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-17 00:46:47,339 INFO    SenderThread:30328 [sender.py:send_exit():597] send defer
2025-03-17 00:46:47,339 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: defer
2025-03-17 00:46:47,340 INFO    HandlerThread:30328 [handler.py:handle_request_defer():172] handle defer: 0
2025-03-17 00:46:47,340 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: defer
2025-03-17 00:46:47,340 INFO    SenderThread:30328 [sender.py:send_request_defer():613] handle sender defer: 0
2025-03-17 00:46:47,341 INFO    SenderThread:30328 [sender.py:transition_state():617] send defer: 1
2025-03-17 00:46:47,341 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: defer
2025-03-17 00:46:47,341 INFO    HandlerThread:30328 [handler.py:handle_request_defer():172] handle defer: 1
2025-03-17 00:46:47,342 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: defer
2025-03-17 00:46:47,342 INFO    SenderThread:30328 [sender.py:send_request_defer():613] handle sender defer: 1
2025-03-17 00:46:47,342 INFO    SenderThread:30328 [sender.py:transition_state():617] send defer: 2
2025-03-17 00:46:47,342 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: defer
2025-03-17 00:46:47,342 INFO    HandlerThread:30328 [handler.py:handle_request_defer():172] handle defer: 2
2025-03-17 00:46:47,342 INFO    HandlerThread:30328 [system_monitor.py:finish():203] Stopping system monitor
2025-03-17 00:46:47,342 DEBUG   SystemMonitor:30328 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2025-03-17 00:46:47,352 DEBUG   SystemMonitor:30328 [system_monitor.py:_start():183] Publishing last batch of metrics
2025-03-17 00:46:47,352 INFO    HandlerThread:30328 [interfaces.py:finish():202] Joined cpu monitor
2025-03-17 00:46:47,354 INFO    HandlerThread:30328 [interfaces.py:finish():202] Joined disk monitor
2025-03-17 00:46:47,436 INFO    HandlerThread:30328 [interfaces.py:finish():202] Joined gpu monitor
2025-03-17 00:46:47,437 INFO    HandlerThread:30328 [interfaces.py:finish():202] Joined memory monitor
2025-03-17 00:46:47,437 INFO    HandlerThread:30328 [interfaces.py:finish():202] Joined network monitor
2025-03-17 00:46:47,438 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: defer
2025-03-17 00:46:47,438 INFO    SenderThread:30328 [sender.py:send_request_defer():613] handle sender defer: 2
2025-03-17 00:46:47,438 INFO    SenderThread:30328 [sender.py:transition_state():617] send defer: 3
2025-03-17 00:46:47,438 DEBUG   SenderThread:30328 [sender.py:send():382] send: stats
2025-03-17 00:46:47,438 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: defer
2025-03-17 00:46:47,439 INFO    HandlerThread:30328 [handler.py:handle_request_defer():172] handle defer: 3
2025-03-17 00:46:47,439 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: defer
2025-03-17 00:46:47,439 INFO    SenderThread:30328 [sender.py:send_request_defer():613] handle sender defer: 3
2025-03-17 00:46:47,439 INFO    SenderThread:30328 [sender.py:transition_state():617] send defer: 4
2025-03-17 00:46:47,439 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: defer
2025-03-17 00:46:47,439 INFO    HandlerThread:30328 [handler.py:handle_request_defer():172] handle defer: 4
2025-03-17 00:46:47,440 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: defer
2025-03-17 00:46:47,440 INFO    SenderThread:30328 [sender.py:send_request_defer():613] handle sender defer: 4
2025-03-17 00:46:47,440 INFO    SenderThread:30328 [sender.py:transition_state():617] send defer: 5
2025-03-17 00:46:47,440 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: defer
2025-03-17 00:46:47,440 INFO    HandlerThread:30328 [handler.py:handle_request_defer():172] handle defer: 5
2025-03-17 00:46:47,441 DEBUG   SenderThread:30328 [sender.py:send():382] send: summary
2025-03-17 00:46:47,442 INFO    SenderThread:30328 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2025-03-17 00:46:47,443 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: defer
2025-03-17 00:46:47,443 INFO    SenderThread:30328 [sender.py:send_request_defer():613] handle sender defer: 5
2025-03-17 00:46:47,443 INFO    SenderThread:30328 [sender.py:transition_state():617] send defer: 6
2025-03-17 00:46:47,444 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: defer
2025-03-17 00:46:47,444 INFO    HandlerThread:30328 [handler.py:handle_request_defer():172] handle defer: 6
2025-03-17 00:46:47,444 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: defer
2025-03-17 00:46:47,444 INFO    SenderThread:30328 [sender.py:send_request_defer():613] handle sender defer: 6
2025-03-17 00:46:47,449 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: status_report
2025-03-17 00:46:47,727 INFO    SenderThread:30328 [sender.py:transition_state():617] send defer: 7
2025-03-17 00:46:47,727 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: defer
2025-03-17 00:46:47,727 INFO    HandlerThread:30328 [handler.py:handle_request_defer():172] handle defer: 7
2025-03-17 00:46:47,728 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: defer
2025-03-17 00:46:47,728 INFO    SenderThread:30328 [sender.py:send_request_defer():613] handle sender defer: 7
2025-03-17 00:46:47,867 INFO    wandb-upload_0:30328 [upload_job.py:push():131] Uploaded file C:\Users\D9110~1.SAI\AppData\Local\Temp\tmpk9j8kduxwandb\x6mkbknv-media/images/Confusion Matrix_10_0922b0f1b30ffc8f26da.png
2025-03-17 00:46:48,215 INFO    Thread-45 :30328 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\config.yaml
2025-03-17 00:46:48,216 INFO    Thread-45 :30328 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\wandb-summary.json
2025-03-17 00:46:48,217 INFO    Thread-45 :30328 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\output.log
2025-03-17 00:46:48,352 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: poll_exit
2025-03-17 00:46:48,870 INFO    wandb-upload_0:30328 [upload_job.py:push():131] Uploaded file C:\Users\D9110~1.SAI\AppData\Local\Temp\tmpk9j8kduxwandb\eznr7ont-media/images/Confusion Matrix_10_0922b0f1b30ffc8f26da.png
2025-03-17 00:46:49,632 INFO    SenderThread:30328 [sender.py:transition_state():617] send defer: 8
2025-03-17 00:46:49,632 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: poll_exit
2025-03-17 00:46:49,633 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: defer
2025-03-17 00:46:49,634 INFO    HandlerThread:30328 [handler.py:handle_request_defer():172] handle defer: 8
2025-03-17 00:46:49,634 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: defer
2025-03-17 00:46:49,634 INFO    SenderThread:30328 [sender.py:send_request_defer():613] handle sender defer: 8
2025-03-17 00:46:49,634 INFO    SenderThread:30328 [sender.py:transition_state():617] send defer: 9
2025-03-17 00:46:49,635 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: defer
2025-03-17 00:46:49,635 INFO    HandlerThread:30328 [handler.py:handle_request_defer():172] handle defer: 9
2025-03-17 00:46:49,635 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: defer
2025-03-17 00:46:49,635 INFO    SenderThread:30328 [sender.py:send_request_defer():613] handle sender defer: 9
2025-03-17 00:46:49,636 INFO    SenderThread:30328 [dir_watcher.py:finish():358] shutting down directory watcher
2025-03-17 00:46:50,240 INFO    SenderThread:30328 [dir_watcher.py:_on_file_modified():288] file/dir modified: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\output.log
2025-03-17 00:46:50,242 INFO    SenderThread:30328 [dir_watcher.py:finish():388] scan: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files
2025-03-17 00:46:50,242 INFO    SenderThread:30328 [dir_watcher.py:finish():402] scan save: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\config.yaml config.yaml
2025-03-17 00:46:50,243 INFO    SenderThread:30328 [dir_watcher.py:finish():402] scan save: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\output.log output.log
2025-03-17 00:46:50,245 INFO    SenderThread:30328 [dir_watcher.py:finish():402] scan save: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\requirements.txt requirements.txt
2025-03-17 00:46:50,246 INFO    SenderThread:30328 [dir_watcher.py:finish():402] scan save: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\wandb-metadata.json wandb-metadata.json
2025-03-17 00:46:50,246 INFO    SenderThread:30328 [dir_watcher.py:finish():402] scan save: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\wandb-summary.json wandb-summary.json
2025-03-17 00:46:50,257 INFO    SenderThread:30328 [dir_watcher.py:finish():402] scan save: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\media\images\Confusion Matrix_10_0922b0f1b30ffc8f26da.png media/images/Confusion Matrix_10_0922b0f1b30ffc8f26da.png
2025-03-17 00:46:50,258 INFO    SenderThread:30328 [sender.py:transition_state():617] send defer: 10
2025-03-17 00:46:50,258 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: defer
2025-03-17 00:46:50,258 INFO    HandlerThread:30328 [handler.py:handle_request_defer():172] handle defer: 10
2025-03-17 00:46:50,259 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: defer
2025-03-17 00:46:50,259 INFO    SenderThread:30328 [sender.py:send_request_defer():613] handle sender defer: 10
2025-03-17 00:46:50,259 INFO    SenderThread:30328 [file_pusher.py:finish():175] shutting down file pusher
2025-03-17 00:46:50,364 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: poll_exit
2025-03-17 00:46:50,364 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: poll_exit
2025-03-17 00:46:50,943 INFO    wandb-upload_1:30328 [upload_job.py:push():131] Uploaded file c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\output.log
2025-03-17 00:46:51,093 INFO    wandb-upload_3:30328 [upload_job.py:push():131] Uploaded file c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\wandb-summary.json
2025-03-17 00:46:51,155 INFO    wandb-upload_2:30328 [upload_job.py:push():131] Uploaded file c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\requirements.txt
2025-03-17 00:46:51,266 INFO    wandb-upload_0:30328 [upload_job.py:push():131] Uploaded file c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\files\config.yaml
2025-03-17 00:46:51,377 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: poll_exit
2025-03-17 00:46:51,377 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: poll_exit
2025-03-17 00:46:51,471 INFO    Thread-44 :30328 [sender.py:transition_state():617] send defer: 11
2025-03-17 00:46:51,472 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: defer
2025-03-17 00:46:51,472 INFO    HandlerThread:30328 [handler.py:handle_request_defer():172] handle defer: 11
2025-03-17 00:46:51,472 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: defer
2025-03-17 00:46:51,472 INFO    SenderThread:30328 [sender.py:send_request_defer():613] handle sender defer: 11
2025-03-17 00:46:51,472 INFO    SenderThread:30328 [file_pusher.py:join():181] waiting for file pusher
2025-03-17 00:46:51,473 INFO    SenderThread:30328 [sender.py:transition_state():617] send defer: 12
2025-03-17 00:46:51,473 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: defer
2025-03-17 00:46:51,473 INFO    HandlerThread:30328 [handler.py:handle_request_defer():172] handle defer: 12
2025-03-17 00:46:51,473 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: defer
2025-03-17 00:46:51,473 INFO    SenderThread:30328 [sender.py:send_request_defer():613] handle sender defer: 12
2025-03-17 00:46:51,473 INFO    SenderThread:30328 [file_stream.py:finish():595] file stream finish called
2025-03-17 00:46:52,320 INFO    SenderThread:30328 [file_stream.py:finish():599] file stream finish is done
2025-03-17 00:46:52,320 INFO    SenderThread:30328 [sender.py:transition_state():617] send defer: 13
2025-03-17 00:46:52,321 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: defer
2025-03-17 00:46:52,321 INFO    HandlerThread:30328 [handler.py:handle_request_defer():172] handle defer: 13
2025-03-17 00:46:52,322 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: defer
2025-03-17 00:46:52,322 INFO    SenderThread:30328 [sender.py:send_request_defer():613] handle sender defer: 13
2025-03-17 00:46:52,323 INFO    SenderThread:30328 [sender.py:transition_state():617] send defer: 14
2025-03-17 00:46:52,324 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: defer
2025-03-17 00:46:52,325 INFO    HandlerThread:30328 [handler.py:handle_request_defer():172] handle defer: 14
2025-03-17 00:46:52,326 DEBUG   SenderThread:30328 [sender.py:send():382] send: final
2025-03-17 00:46:52,326 DEBUG   SenderThread:30328 [sender.py:send():382] send: footer
2025-03-17 00:46:52,326 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: defer
2025-03-17 00:46:52,327 INFO    SenderThread:30328 [sender.py:send_request_defer():613] handle sender defer: 14
2025-03-17 00:46:52,330 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: poll_exit
2025-03-17 00:46:52,332 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: poll_exit
2025-03-17 00:46:52,339 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: internal_messages
2025-03-17 00:46:52,342 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: server_info
2025-03-17 00:46:52,343 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: get_summary
2025-03-17 00:46:52,344 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: sampled_history
2025-03-17 00:46:52,344 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: server_info
2025-03-17 00:46:52,346 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: job_info
2025-03-17 00:46:52,612 DEBUG   SenderThread:30328 [sender.py:send_request():409] send_request: job_info
2025-03-17 00:46:52,620 DEBUG   HandlerThread:30328 [handler.py:handle_request():146] handle_request: shutdown
2025-03-17 00:46:52,620 INFO    HandlerThread:30328 [handler.py:finish():866] shutting down handler
2025-03-17 00:46:53,357 INFO    WriterThread:30328 [datastore.py:close():294] close: c:\Users\D. Sai Mani Kumar\Downloads\DL\da6401_assignment1\wandb\run-20250317_004212-qh0l59uo\run-qh0l59uo.wandb
2025-03-17 00:46:53,625 INFO    SenderThread:30328 [sender.py:finish():1548] shutting down sender
2025-03-17 00:46:53,625 INFO    SenderThread:30328 [file_pusher.py:finish():175] shutting down file pusher
2025-03-17 00:46:53,625 INFO    SenderThread:30328 [file_pusher.py:join():181] waiting for file pusher
